resolvers/webpack/index.js

```diff
@@ -293,17 +292,20 @@ const MAX_CACHE = 10;
 const _cache = [];
 function getResolveSync(configPath, webpackConfig, cwd) {
   const cacheKey = { configPath, webpackConfig };
-  let cached = find(_cache, function (entry) { return isEqual(entry.key, cacheKey); });
-  if (!cached) {
-    cached = {
-      key: cacheKey,
-      value: createResolveSync(configPath, webpackConfig, cwd),
-    };
-    // put in front and pop last item
-    if (_cache.unshift(cached) > MAX_CACHE) {
-      _cache.pop();
+  for (const entry of _cache) {
```

Comment:
this uses for-of tho, which is slower than standard array iteration - a standard for loop would be the replacement here

Reply:
2c577cb  can be undone if plain `for` loops are preferable, but I see both `for-of` and plain `for` loops used everywhere in the repo. I'd say `for-of` gives a good balance between performance (still better than polyfilling .find()) and readability (over plain `for` loops)

Reply:
Let me know if I should revert `2c577cb` or if for-of is good enough here

Reply:
Done, it uses a regular for loop now